src/translations/i18n.ts

```diff
@@ -4,6 +4,44 @@ import enTranslations from './enTranslations.json';
 import nbTranslations from './nbTranslations.json';
 import nnTranslations from './nnTranslations.json';
 
+export const samiLanguageCodes = [
+  'smi', // General for all sami languages
+  'se', // Northern Sami
+  'sme', // Northern Sami
+  'sma', // Southern Sami
+  'smj', // Lule Sami
+  'smn', // Inari Sami
+  'sms', // Skolt Sami
+  'sje', // Pite Sami
+  'sju', // Ume Sami
+  'sjd', // Kildin Sami
+  'sjt', // Ter Sami
+];
+
+const norwegianBokmaalLanguages = ['nob', 'nb', 'nb-NO', 'no', 'no-NO'];
+const norwegianNynorskLanguages = ['nn', 'nno', 'nn-NO'];
+const englishLanguages = ['eng', 'en', 'en-US', 'en-GB'];
+
+const handledLanguages = [
+  ...englishLanguages,
+  ...norwegianBokmaalLanguages,
+  ...norwegianNynorskLanguages,
+  ...samiLanguageCodes,
+];
+
+export const selectDisplayLanguage = (langCode: string | undefined | null) => {
+  // Might be a string because it might come from a cookie
+  if (langCode === 'undefined' || langCode === 'null' || langCode === undefined || langCode === null) {
```

Comment:
Med `!langCode` kunne vi fanga både `null`, `undefined`, og `''`
```suggestion
  if (langCode === 'undefined' || langCode === 'null' || !langCode) {
```